keyboards/bioi/g60ble/rules.mk

```diff
@@ -0,0 +1,29 @@
+# MCU name
+MCU = atmega32u4
+F_CPU = 8000000
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = qmk-dfu
```

Comment:
Does the board actually run qmk-dfu by default?

Reply:
lol, um... when I'm in the bootloader mode, I see QMK referenced in the dmesg output, so I _think_ so, but am not sure how to otherwise tell.

Reply:
You'd have to flash the bootloader after compiling it yourself. 

Otherwise, you likely have ATMEL DFU on the board.  Those are the only bootloaders that come factory. 

However, if you're using a fab that is preloading the firmware, and used the `:production` target, then it's using QMK DFU.

Reply:
I'm not the one producing the board. All I can tell you is that, while watching dmesg output, switching to the (stock from the factory) bootloader, I see qmk referenced.

Reply:
Could you post the dmesg output here? I'm curious what it says exactly.